packages/adapter/src/utils.ts

```diff
@@ -105,11 +105,11 @@ export const isLocked = async (): Promise<boolean> => {
 /**
  * Prompt the user to allow the snap
  */
-export const enableShapeShiftSnap = async (
+export const enableShapeShiftMultichain = async (
```

Comment:
Ditto consumed with the current name in hdwallet PR, revert

Reply:
see above

Reply:
5270461df8d5125867707dd9d582f3c7a6ce5aae